Automattic/wp-calypso
# Social Previews: Update package to support titleless posts

Related to https://github.com/Automattic/jetpack/pull/35728

## Proposed Changes

* Updated each preview to support our new titleless Social Notes format.

## Testing Instructions



This cannot be tested in Calypso natively as the previews for social posts live in jetpack under wp-admin.

Test together with this change: https://github.com/Automattic/jetpack/pull/35728

For sanity testing, you can check that in Calypso under the Posts menu if you check the previews, it works as intended.

## Pre-merge Checklist
